scripts/20210421_has-allowed-email-domains/cleanup.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable */
+
+// Optional: Retrieve forms with fields that has a populated allowedEmailDomains but has no allowed email domains.
+db.getCollection('forms').aggregate([
+  { $unwind: '$form_fields' },
+  {
+    $match: {
+      'form_fields.fieldType': 'email',
+      'form_fields.hasAllowedEmailDomains': false,
+      'form_fields.allowedEmailDomains': { $ne: [] },
+    },
+  },
+  {
+    $group: {
+      _id: '$_id',
+      form_fields: { $addToSet: '$form_fields' },
+    },
+  }
+])
+
+// Retrieve counts of various form states
+// Count forms with email fields isVerifiable=true, hasAllowedEmailDomains=true, non-empty allowedEmailDomains
+// Basically forms with restricted domain email fields
+
+// COUNT_A =
+
+db.getCollection('forms').count({
+  'form_fields.fieldType': 'email',
+  'form_fields.isVerifiable': true,
+  'form_fields.hasAllowedEmailDomains': true,
+  'form_fields.allowedEmailDomains': { $ne: [] },
```

Comment:
we need an `$elemMatch` here right? otherwise this will pick up e.g. two separate email fields, one with isVerifiable true and one with hasAllowedEmailDomains true, but neither of which satisfies all the criteria